### ISSUE TITLE
License compliance check will fail soon

### ISSUE THREAD

**Issue by Geal** [apollographql/router#295]
apparently cargo-about cannot detect the license for encoding_rs, when running `cargo update` I get this:

```
error[L003]: encoding_rs = 0.8.30 is unlicensed                                                  
  ┌─ encoding_rs 0.8.30 (registry+https://github.com/rust-lang/crates.io-index):2:9              
  │                                                                                              
2 │ name = "encoding_rs"                                                                         
  │         ^^^^^^^^^^^ a valid license expression could not be retrieved for the crate          
3 │ version = "0.8.30"                                                                           
4 │ license = ""                                                                                 
  │            - license expression was not specified                                            
5 │ license-files = [                                                                            
6 │     { path = "COPYRIGHT", hash = 0x39f8ad31, score = 0.14 },                                 
  │                                                      ---- low confidence in the license text
  │                                                                                              
  = encoding_rs v0.8.30                                                                          
    ├── isahc v1.6.0                                                                             
    │   └── httpmock v0.6.5                                                                      
    │       └── (dev) apollo-router v0.1.0-alpha.2                                               
    │           └── (dev) apollo-router-benchmarks v0.1.0-alpha.2                                
    └── reqwest v0.11.8                                                                          
        ├── (dev) apollo-router v0.1.0-alpha.2 (*)                                               
        ├── apollo-router v0.1.0-alpha.2 (*)                                                     
        ├── reqwest-middleware v0.1.3                                                            
        │   ├── apollo-router v0.1.0-alpha.2 (*)                                                 
        │   └── reqwest-tracing v0.2.0                                                           
        │       └── apollo-router v0.1.0-alpha.2 (*)                                             
        ├── reqwest-tracing v0.2.0 (*)                                                           
        └── xtask v0.1.0-prealpha.1                                                              
```

**Comment by Geal** [comment]
probably linked to https://github.com/EmbarkStudios/cargo-about/issues/152

**Comment by Geal** [comment]
this has been fixed in #281
